src/app/shared/pipes/userDisplay.ts

```diff
@@ -6,17 +6,19 @@ export interface FormattableUser {
   lastName: string|null,
 }
 
+export function formatUser<T extends FormattableUser>(userInfos: T) : string {
+  if (userInfos.firstName !== null && userInfos.firstName !== '' && userInfos.lastName !== null && userInfos.lastName !== '') {
+    return `${userInfos.firstName} ${userInfos.lastName} (${userInfos.login})`;
+  } else if (userInfos.firstName !== null && userInfos.firstName !== '') {
+    return `${userInfos.firstName} (${userInfos.login})`;
+  } else if (userInfos.lastName !== null && userInfos.lastName !== '') {
+    return `${userInfos.lastName} (${userInfos.login})`;
+  } else {
+    return userInfos.login;
+  }
+}
+
```

Comment:
I would prefer you move this function out of the pipe. For instance, in `helpers`.

Reply:
fixed